featuretools/computational_backends/pandas_backend.py

```diff
@@ -98,50 +98,11 @@ def calculate_all_features(self, instance_ids, time_last,
             ordered_entities = self.feature_tree.ordered_entities
 
         necessary_columns = self.feature_tree.necessary_columns
-        eframes_by_filter = \
-            self.entityset.get_pandas_data_slice(filter_entity_ids=ordered_entities,
-                                                 index_eid=self.target_eid,
-                                                 instances=instance_ids,
-                                                 entity_columns=necessary_columns,
-                                                 time_last=time_last,
-                                                 training_window=training_window,
-                                                 verbose=verbose)
-        large_eframes_by_filter = None
+        large_necessary_columns = None
         if any([f.primitive.uses_full_entity
                 for f in self.feature_tree.all_features
                 if isinstance(f, (GroupByTransformFeature, TransformFeature))]):
             large_necessary_columns = self.feature_tree.necessary_columns_for_all_values_features
-            large_eframes_by_filter = \
-                self.entityset.get_pandas_data_slice(filter_entity_ids=ordered_entities,
-                                                     index_eid=self.target_eid,
-                                                     instances=None,
-                                                     entity_columns=large_necessary_columns,
-                                                     time_last=time_last,
-                                                     training_window=training_window,
-                                                     verbose=verbose)
-
-        # Handle an empty time slice by returning a dataframe with defaults
-        if eframes_by_filter is None:
-            return self.generate_default_df(instance_ids=instance_ids)
```

Comment:
We can't really check this condition anymore and I wasn't sure if this should be moved somewhere else so I just got rid of it. The tests still pass, so maybe it's not necessary?